src/Gateway/Events/ThreadMemberUpdate.php

```diff
@@ -17,7 +18,7 @@ class ThreadMemberUpdate
     public int $member_count;
 
     /**
-     * @var \Ragnarok\Fenrir\Parts\ThreadMember[]
+     * @var ThreadMember[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)